docs/general/administration/hardware-selection.md

```diff
@@ -114,9 +114,9 @@ Encoder Quality: Apple ≥ Intel ≥ Nvidia >>> AMD<sup>\*</sup>
 Intel is always recommended on non-apple hardware for the following reasons:
 
 - Intel provides a good quality encoder, slightly better than Nvidia and significantly better than AMD.
-- Intel drivers and the compute environment is much easier to setup than both Nvidia and AMD
+- Intel drivers and the compute environment are much easier to setup than both Nvidia and AMD
 
-AMD is not recommended because of poor quality H.264 and H.265(HEVC) output, as well as being hard to setup the compute environment. While AMD has significantly improved AV1 encoder quality. However you are still more likely to transcode to H.264 or H.265 than to AV1 due to compatibility.
+AMD is not recommended because of poor quality H.264 and H.265(HEVC) output, as well as being hard to setup the compute environment. While AMD has significantly improved AV1 encoder quality. However, you are still more likely to transcode H.264 or H.265 than AV1 due to compatibility.
```

Comment:
```suggestion
AMD is not recommended because of poor quality H.264 and H.265(HEVC) output, as well as being hard to setup the compute environment. While AMD has significantly improved AV1 encoder quality. However, you are still more likely to transcode to H.264 or H.265 than AV1 due to compatibility.
```
I think in this case we can't omit it because it adds clarity